core/components-common/src/main/java/org/talend/components/common/tableaction/TableActionConfig.java

```diff
@@ -50,6 +50,8 @@
     public boolean SQL_CREATE_TABLE_DEFAULT_ENABLED = true;
     public String SQL_CREATE_TABLE_IF_NOT_EXISTS = "IF NOT EXISTS";
 
+    public String SQL_CREATE_TABLE_NOT_NULL = " NOT NULL";

```

Comment:
you have a trilling space here, is it normal ?
```suggestion
    public String SQL_CREATE_TABLE_NOT_NULL = "NOT NULL";
```